buildscripts/libs/build_nccmp.sh

```diff
@@ -41,8 +41,8 @@ else
     export CXX=$SERIAL_CXX
 fi
 
-export CFLAGS="-fPIC"
-export LDFLAGS="-L$NETCDF_ROOT/lib -L$HDF5_ROOT/lib -L$SZIP_ROOT/lib"
+export CFLAGS="-fPIC ${JEDI_CFLAGS}"
+export LDFLAGS="-L$NETCDF_ROOT/lib -L$HDF5_ROOT/lib -L$SZIP_ROOT/lib ${JEDI_LDFLAGS}"
 
 url="https://gitlab.com/remikz/nccmp/-/archive/$version/${software}.tar.gz"
```

Comment:
Looks like this tool's git structure has changed _archive_ to _tree_
```suggestion
url="https://gitlab.com/remikz/nccmp/-/tree/$version/${software}.tar.gz"
```